Automattic/pocket-casts-ios
# EOY 2023: Intro Story

| 📘 Part of: #1142 | 🎨 Designs: [Figma](https://www.figma.com/file/lH66LwxxgG8btQ8NrM0ldx/End-of-Year?type=design&node-id=1599-20385&mode=design) |
|:---:|:---:|

Adds the 2023 visual for the Intro Story.

https://github.com/Automattic/pocket-casts-ios/assets/7040243/d812ec71-b45c-4c7f-bee8-e98284ac1555

## To test

Please, test on a real device so you can test the parallax effect.

1. Make sure you're logged in into an account that has a few episodes listened
2. Go to Profile > Settings > Beta Features > enable `endOfYear`
3. Go back to Profile
4. Tap the EOY image
5. ✅ Check that the Intro Story looks good
6. Tilt your device
7. ✅ The "2023" should move around

Please test on a big device (iPhone 15 Pro Max, for example), normal device (iPhone 15) and smaller device (iPhone SE).

## Discussion

There's a small issue when going back to the story in which the 2023 moves it's position quite abruptly. To test, Move to the second story, then go back to the first one. I'm still not sure how to prevent or fix that so I'll add a note about in on the Project Issue.

## Checklist
